content/en/references/network-troubleshooting/endpoint-url/_index.md

```diff
@@ -0,0 +1,137 @@
+docker network create my-network
+# launch localstack
+LAMBDA_DOCKER_NETWORK=my-network DOCKER_FLAGS="--network my-network" localstack start
+# then your code can access localstack at its container name (by default: localstack_main)
+aws --endpoint-url http://localstack_main:4566 s3api list-buckets
+{{</tab>}}
+{{<tab header="Docker" lang="bash">}}
+# create the network
+docker network create my-network
+# launch localstack
+docker run --rm -it --network my-network -e LAMBDA_DOCKER_NETWORK=my-network <other flags> localstack/localstack[-pro]
+# then your code can access localstack at its container name (by default: localstack_main)
+aws --endpoint-url http://localstack_main:4566 s3api list-buckets
+{{</tab>}}
+{{<tab header="docker-compose.yml" lang="yml">}}
+services:
+  localstack:
+    # ... other configuration here
+    environment:
+      LAMBDA_DOCKER_NETWORK=ls
+    networks:
+    - ls
+networks:
+  ls:
+    name: ls
+
+# Your application code can then use
+# http://localstack:4566 for the
+# endpoint url
+{{</tab>}}
+{{</tabpane>}}
+
+# From your container
+
+{{< figure src="../images/7.svg" width="400" >}}
+
+**Example**: you are running your application code in a container and accessing AWS resources such as S3 in LocalStack.
+
+It may be useful to start LocalStack in a [user-defined network](https://docs.docker.com/network/bridge/), and set the `LAMBDA_DOCKER_NETWORK` environment variable to this value.
+Then your application code can access the LocalStack instance by its hostname. For example:
+
+{{<tabpane>}}
+{{<tab header="CLI" lang="bash">}}
+# create the network
+docker network create my-network
+# launch localstack
+DOCKER_FLAGS="--network my-network" localstack start
+# launch your container
+docker run --rm it --network my-network <image name>
+# then your code can access localstack at its container name (by default: localstack_main)
+{{</tab>}}
+{{<tab header="Docker" lang="bash">}}
+# create the network
+docker network create my-network
+# launch localstack
+docker run --rm -it --network my-network <other flags> localstack/localstack[-pro]
+# launch your container
+docker run --rm it --network my-network <image name>
+# then your code can access localstack at its container name (by default: localstack_main)
+{{</tab>}}
+{{<tab header="docker-compose.yml" lang="yml">}}
+services:
+  localstack:
+    # ... other configuration here
+    networks:
+    - ls
+  your_container:
+    # ... other configuration here
+    networks:
+    - ls
+networks:
+  ls:
+    name: ls
+
+# Your application code can then use
+# http://localstack:4566 for the
+# endpoint url
+{{</tab>}}
+{{</tabpane>}}
+
+# From a separate host
+
+{{< figure src="../images/10.svg" width="400" >}}
+
+LocalStack must listen to the address of the host, or `0.0.0.0`.
+
+{{<tabpane>}}
+{{<tab header="CLI" lang="bash">}}
+EDGE_BIND_HOST="0.0.0.0" localstack start
+{{</tab>}}
+{{<tab header="Docker" lang="bash">}}
+# this command exposes ports on 0.0.0.0 by default
```

Comment:
We should rephrase this a bit, as it can be seen as factually wrong. It binds on all interfaces, both ipv4 and ipv6. 
Example:
```
{
                "4566/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "4566"
                    },
                    {
                        "HostIp": "::",
                        "HostPort": "4566"
                    }
                ]
            }
```
Maybe we can just rephrase it as `this command exposes ports on all interfaces` or something.